build.rs

```diff
@@ -1,46 +1,51 @@
-//! Reads `template/index.ron` and includes the files listed their into the source code.
```

Comment:
I think such a description is quite helpful, so please don't remove it but rather change it.